src/dialects/mariadb/index.js

```diff
@@ -2,13 +2,13 @@
 
 const _ = require('lodash');
 const { AbstractDialect } = require('../abstract');
-const ConnectionManager = require('./connection-manager');
-const Query = require('./query');
-const QueryGenerator = require('./query-generator');
-const { MySQLQueryInterface } = require('../mysql/query-interface');
+const { MariaDbConnectionManager } = require('./connection-manager');
+const { MariaDbQuery } = require('./query');
+const { MariaDbQueryGenerator } = require('./query-generator');
+const { MySqlQueryInterface } = require('../mysql/query-interface');
```

Comment:
Interesting, why does this not use a MariaDbQueryInterface? Might be good to add in a future PR so it's easier when we split dialects up

Reply:
I assume it's because both dialects are very similar (mariadb being a fork of mysql), it might not be necessary to reimplement one for mariadb.

Once we split the dialects, the mysql dialect can still be a dependency of the mariadb one